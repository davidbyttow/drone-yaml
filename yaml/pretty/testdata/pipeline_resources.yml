kind: pipeline
name: default

steps:
  - name: build
    image: golang
    commands:
      - go build
    resources:
      limits:
        cpu: 200m
        memory: '100Mi'
      requests:
        cpu: 100m
        memory: '50Mi'
