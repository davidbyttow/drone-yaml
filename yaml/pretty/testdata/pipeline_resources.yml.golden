---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: build
  image: golang
  commands:
  - go build
  resources:
    limits:
      cpu: 200m
      memory: 100Mi
    requests:
      cpu: 100m
      memory: 50Mi

...
